src/System.Private.Reflection.Metadata.Ecma335/dir.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.1.3.0</AssemblyVersion>
```

Comment:
Nit: can we also use this PR to remove the AssemblyVersion on the actual csproj?